GameData/KerbalismConfig/Support/ROCapsules.cfg

```
@@ -585,8 +985,9 @@
 {
 	@MODULE[Habitat]
 	{
-		%volume = 16.8	//According to ETS wiki
-		%surface = 33.0	//guesstimate
+		%volume = 20.8	//m^3 ~ Guess
```

```suggestion
		%volume = 16.8	//According to ETS wiki
```
The wiki had the mission module IV as having the same habitable volume, but with extra cargo space.